wpilibc/sim/src/SampleRobot.cpp

```diff
@@ -18,7 +18,7 @@
 #include <unistd.h>
 #elif defined(_WIN32)
 #include <windows.h>
-void sleep(unsigned milliseconds) { Sleep(milliseconds); }
+void sleep(int milliseconds) { Sleep(milliseconds); }
```

Comment:
This shouldn't be changed (this is a compatibility shim and the Unix sleep function takes an unsigned parameter).